packages/uniforms-bootstrap3/__tests__/_createContext.ts

```diff
@@ -28,8 +29,8 @@ export default function createContext(
         placeholder: false,
         readOnly: false,
         showInlineError: false,
-        ...context?.state,
-      },
-    },
+        ...context?.state
+      }
+    }
```

Comment:
prettier config might have changed. it says `trailingComma: "none"`, so i guess the reformat is correct, but i am unsure